javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,75 @@
+// Find the shortest word
+console.log("---------------------- 1st Task - Find the shortest word ----------------------");
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function findShortestWord(list) {
+  let shortestWordLength = 100,
+    shortestWordIndex;
+  for (var i = 0; i < list.length; i++) {
+    if (shortestWordLength > list[i].length) {
+      shortestWordLength = list[i];
+      shortestWordIndex = i;
+    }
+  }
+  return list[shortestWordIndex];
+}
+console.log("The shortest word in the list is: " + findShortestWord(danishWords)); // returns 'ø'
+
+// Find and count the Danish letters
+console.log("---------------------- 2nd Task - Find Danish letters count ----------------------");
+const danishString1 = "Jeg har en blå bil";
+const danishString2 = "Blå grød med røde bær";
+let danishLettersCount = [0, 0, 0];
+let danishLettersCountTotal = 0;
+
+function findDanishLettersCount(list) {
+  for (var i = 0; i < list.length; i++) {
+    let checkWord = list[i];
+    for (var j = 0; j < checkWord.length; j++) {
+      isDanishLetterFound(checkWord[j]);
+    }
+  }
+  return ("{total: " + danishLettersCountTotal + ", æ: " + danishLettersCount[0] + ", ø: " + danishLettersCount[1] + ", å: " + danishLettersCount[2] + "}");
+}
+
+function isDanishLetterFound(letter) {
```

Comment:
Good naming convention, but remember that usually when you have a `isSomething` method you would expect it to return either true or false